main.css

```
@@ -2,4 +2,15 @@
     margin: 0;
     padding: 0;
     box-sizing: border-box;
-}
\ No newline at end of file
+}
+
+body{
+    font-family: Arial;
+    background-color:#FFCB77;
```

🙃 space for declarations
```suggestion
    background-color: #FFCB77;
```